README.md

```diff
@@ -111,6 +111,18 @@ run:
 ```
 opam install bap-ida-python
 ```
+After the installation you'll see few commands that you need to run, e.g.:
+```
+=> In order to install bap-ida-python plugin:
+$: rm -rf /home/bap-user/ida/plugins/bap/
```

Comment:
could you please remove this `$:`  not only they don't look like a prompt, but they also prevent users from copying this commands at once. 

Also, to make these commands usable, change `/home/bap-user/.opam/share` to `$(opam var prefix)` and `/home/bap-user/` to `$IDA_PATH` with corresponding comment that `IDA_PATH` denotes the root of IDA Pro installation.